shared/bash_remediation_functions/substitute_in_pam_file.sh

```diff
@@ -0,0 +1,32 @@
+function substitute_in_pam_file {
```

Comment:
This function is specific for configuring `pam_faillock` module, it should be named according to it.